### PR TITLE
Add kubectl-kount plugin 

### PR DESCRIPTION
**Pull request by adityajoshi12**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Review by ahmetb — CHANGES_REQUESTED** [review]
sorry forgot to publish comments.

File: plugins/kount.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          arch: amd64
+      uri: https://github.com/adityajoshi12/kubectl-kount/releases/download/v0.0.1/kubectl-kount_v0.0.1_windows_amd64.tar.gz
+      sha256: 30f16adcca489e1cd25cde8e5bd67d505219263868d3653eb0381f1eaaf6e47c
+      bin: kubectl-kount.exe
+      files:
+        - from: kubectl-kount.exe
+          to: .
+        - from: LICENSE
+          to: .
+  shortDescription: kount Kubernetes resource instances.
```
[the remark this answers is not in the document]
Comment (adityajoshi12) [reply plugins/kount.yaml]: updated the shortDescription.

File: plugins/kount.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  description: |
+    kubectl-kount is a subcommand of kubectl to to kount kubernetes object.
```
[the remark this answers is not in the document]
Comment (adityajoshi12) [reply plugins/kount.yaml]: Added more details to description

**Comment by ahmetb** [comment]
I've just had a chance to use the plugin and I'm afraid it doesn't meet the usefulness criteria. It doesn't do anything more interesting than `kubectl get -n NAMESAPCE RESOURCE_TYPE | wc -l`. We typically reject plugins that are easily achievable with such one-liner commands.

**Comment by adityajoshi12** [comment]
This is just the initial release, next release is going to have have more features like detailed view and group by namespaces.

**Comment by ahmetb** [comment]
Understood, but you should consider submitting the plugin when that becomes the case. As it stands right now, the plugin doesn't meet the usefulness criteria. Also, I recommend you to think more broadly in terms of querying/grouping objects besides just counting. In that case, "kount" name no longer becomes useful either. So please take more time developing your plugin, which may help you better understand what you want to build and what to name it.

**Pull request closed without merging**